test/vscode/integration_tests/binaries_and_tests_test.rs

```diff
@@ -0,0 +1,62 @@
+#[cfg(test)]
+mod tests {
+    use std::env;
+    use std::fs;
+    use std::path::PathBuf;
+
+    #[test]
+    fn test_binaries_and_tests() {
+        let launch_json_path = PathBuf::from(env::var("LAUNCH_JSON").unwrap());
+        let content = fs::read_to_string(&launch_json_path)
+            .unwrap_or_else(|_| panic!("couldn't open {:?}", &launch_json_path));
+
+        // Verify basic JSON structure
+        assert!(content.contains(r#""version": "0.2.0""#));
```

Comment:
I'd be tempted to actually parse the JSON here rather than doing string asserts, but also recognise going from 0 to non-0 dependencies is annoying

Reply:
Done